lib/integrations/front.ts

```diff
@@ -983,6 +983,22 @@ async function getConversationChannel(
 		});
 	});
 
+	const lastMessageId = conversationResponse._links.related.last_message
+		.split('/')
+		.pop()
+		.split('?')[0];
```

Comment:
Since you only care about the first element you can limit `split` to just 1 match and avoid any unnecessary work:
```suggestion
		.split('?', 1)[0];
```